### PR TITLE
Replace logo viewlet with ftw.logo

### PR DESCRIPTION
**Pull request by busykoala**
Removing the logo viewlet enables ftw.logo to add its logo in a more consistent way.

### CODE REVIEW

File: plonetheme/onegov/upgrades/20190703161348_install_ftw_logo/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<metadata>
+  <dependencies>
+    <dependency>profile-ftw.logo:default</dependency>
```
Comment (maethu) [inline plonetheme/onegov/upgrades/20190703161348_install_ftw_logo/metadata.xml]: We don't need both. Either the metadata.xml, or `self.setup_install_profile('profile-ftw.logo:default')` in the upgrade.py. I suggest you delete that one. Now the ftw.logo gets installed twice 😊

Comment (mbaechtold) [reply plonetheme/onegov/upgrades/20190703161348_install_ftw_logo/metadata.xml]: I strongly recommend to use `ftw.upgrade.step.UpgradeStep#ensure_profile_installed` in order to install Generic Setup Profiles in an Upgrade Step. And never use the `metadata.xml` in the UpgradeStep.

File: plonetheme/onegov/viewlets/configure.zcml [hunk 2]
```diff
@@ -11,23 +11,6 @@
...
-      manager="plone.app.layout.viewlets.interfaces.IPortalHeader"
-      class=".logo.LogoViewlet"
-      permission="zope2.View"
-      layer="plonetheme.onegov.interfaces.IPlonethemeOneGovLayer"
-      />
-
-  <browser:viewlet
-      name="plone.links.favicon"
-      manager="plone.app.layout.viewlets.interfaces.IHtmlHeadLinks"
-      class=".meta.MetaViewlet"
```
Comment (mbaechtold) [inline plonetheme/onegov/viewlets/configure.zcml]: Could we also remove the file `plonetheme/onegov/viewlets/meta.py`?

File: plonetheme/onegov/viewlets/configure.zcml [hunk 3]
```diff
@@ -11,23 +11,6 @@
       layer="plonetheme.onegov.interfaces.IPlonethemeOneGovLayer"
       />
 
-  <!-- The logo -->
-  <browser:viewlet
-      name="plone.logo"
-      manager="plone.app.layout.viewlets.interfaces.IPortalHeader"
-      class=".logo.LogoViewlet"
```
Comment (mbaechtold) [inline plonetheme/onegov/viewlets/configure.zcml]: Could we also remove the file `plonetheme/onegov/viewlets/logo.py`?

Comment (mbaechtold) [reply plonetheme/onegov/viewlets/configure.zcml]: Already done 👍

File: plonetheme/onegov/viewlets/configure.zcml (hunk 2 above)
Comment (mbaechtold) [reply plonetheme/onegov/viewlets/configure.zcml]: And the file `plonetheme/onegov/viewlets/meta.pt` too?

File: plonetheme/onegov/upgrades/20190703161348_install_ftw_logo/metadata.xml (hunk 1 above)
Comment (busykoala) [reply plonetheme/onegov/upgrades/20190703161348_install_ftw_logo/metadata.xml]: 👍

File: plonetheme/onegov/viewlets/configure.zcml (hunk 2 above)
Comment (busykoala) [reply plonetheme/onegov/viewlets/configure.zcml]: 👍

**Pull request merged**